src/game_battlealgorithm.cpp

```diff
@@ -776,7 +781,7 @@ bool Game_BattleAlgorithm::Normal::Execute() {
 
 	// Damage calculation
 	if (Utils::GetRandomNumber(0, 99) < to_hit) {
-		if (!source->IsCharged() && Utils::GetRandomNumber(0, 99) < (int)ceil(crit_chance * 100)) {
+		if (Utils::GetRandomNumber(0, 99) < (int)ceil(crit_chance)) {
```

Comment:
Should either use floats or a multiplier for the integer (RPG_RT uses 10000 instead of 100 as far as I know, since you may have crit chances of less than 1%)

Reply:
I just changed that out before you commented. The float returned is now scaled 0-100.0f and the `* 100` here is gone.

Reply:
Okay correction, less than 1% normally not because the box is capped to "1 in 100", but still there would otherwise be no difference between "1 in 100" and "1 in 90" for example.

Reply:
Wait I think I see what you mean. This algo will always clamp to a minimum of 1%, which is not great.